app/views/rounds/index.html.erb

```diff
@@ -0,0 +1,33 @@
+<h1>Hello World!</h1>
+
+<table class='rounds-table'>
+  <thead>
+    <tr>
+      <th class="rounds-table-title"colspan="8">All Rounds</th>
+    </tr>
+    <tr>
+      <th class="round-data-type">Holes</th>
+      <th class="round-data-type">Course</th>
+      <th class="round-data-type">Date</th>
+      <th class="round-data-type">Par</th>
+      <th class="round-data-type">Score</th>
+      <th class="round-data-type">Difficulty</th>
+      <th class="round-data-type">Drinking</th>
+      <th class="round-data-type">Comments</th>
```

Comment:
Rather than adding a class to all of these, you could add a class to the parent, and then target these `th` elements as `tr.round-data-headers th` in CSS